templates/landscape/managed.html

```diff
@@ -0,0 +1,157 @@
+        <p class="p-heading--2">24/7 <br> monitoring</p>
+        <p>Active management and break/fix response by Canonical engineers</p>
+      </div>
+      <div class="col-medium-2 col-3">
+        <p class="p-heading--2">High <br> availability</p>
+        <p>Fault-tolerant and ready for real-world challenges</p>
+      </div>
+    </div>
+  </section>
+  
+  <section class="p-strip is-deep u-no-padding--top">
+    <hr class="u-no-margin--bottom is-fixed-width u-hide--small u-hide--medium">
+    <div class="row">
+      <div class="col-3">
+        <h2 class="p-muted-heading">Why <br class="u-hide--small u-hide--medium"> Managed Landscape</h2>
+        <hr class="is-fixed-width u-no-margin--bottom u-hide--large">
+
+      </div>
+      <div class="col-9">
+        <div class="p-strip u-no-padding--top is-shallow">
+          <h2>Any-cloud ready</h2>
+        </div>
+        <p>Managed Landscape can be deployed in high availability across multiple machines, anywhere. Whether you are running your workloads across multiple clouds or a hybrid cloud with your own infrastructure, Managed Landscape can easily provide a scalable and reliable service to maintain your Ubuntu environment.</p>
+      </div>
+    </div>
+      <div class="p-strip is-shallow u-no-padding--top">
+        <div class="row">
+          <div class="col-small-1 col-2 col-start-large-2  col-medium-1 u-sv-3"><img src="https://assets.ubuntu.com/v1/38ac7357-openstack-logo.png" alt="OpenStack"></div>
+          <div class="col-small-1 col-2 col-medium-1 u-sv-3"><img src="https://assets.ubuntu.com/v1/865293a8-microsoft-azure--logo.png" alt="Microsoft Azure"></div>
+          <div class="col-small-1 col-2 col-medium-1 u-sv-3"><img src="https://assets.ubuntu.com/v1/b2952436-aws-logo.svg" alt="Amazon Web Services"></div>
+          <div class="col-small-1 col-2 col-medium-1 u-sv-3"><img src="https://assets.ubuntu.com/v1/8347ea7c-vmware-logo.png" alt="VMWare"></div>
+          <div class="col-small-1 col-2 col-medium-1 u-sv-3"><img src="https://assets.ubuntu.com/v1/120a5c4b-google-cloud-stacked-logo.png" alt="Google Cloud Platform"></div>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-9 col-start-large-4">
+          <div class="p-strip u-no-padding--top">
+            <hr class="is-fixed-width u-no-margin--bottom ">
+            <div class="p-strip u-no-padding--top is-shallow">
+              <h2 class="Why Managed Landscape">Bring your own <br class="u-hide--small"> identity and access management platform</h2>
+            </div>
+              <p>Landscape supports OpenID Connect (OIDC) to authenticate users, and integrates enterprise identity providers like Azure AD, Google, Okta, IBM, and more.</p>
+          </div>
+          <hr class="u-no-margin--bottom">
+          <div class="p-strip u-no-padding--top is-shallow">
+            <h2>Mirror Canonical's repositories. <br class="u-hide--small"> Host private repositories</h2>
```

Comment:
It seems weird that we're not punctuating any other like-sized heading with periods except for this one. I've pinged the pdm directly asking if I can change it since google docs won't let me add screenshots to comments I guess 😅

Reply:
Change +1ed by pdm, I've updated the doc accordingly